deployments/k3d/Makefile

```diff
@@ -1,50 +1,59 @@
 IMAGE_NAME ?= k3swithshims
 CLUSTER_NAME ?= k3s-default
+PLATFORM ?= linux/amd64
```

Comment:
I found this configuration option useful when working with MacOS M1. Now, I can configure the image to `linux/arm64` when working with k3d. However, I can move this to a separate PR if it's more convenient